### ISSUE TITLE
Plotting gas traces with matplotlib plot title disappears

### ISSUE THREAD

**Issue by belasebok** [SpectroInlets/cinfdata#3]
If you put in a title in matplotlib options and try to plot, it doesn't show up on the graph and disappears from the field on the homepage.

**Comment by KennethNielsen** [comment]
I don't think we support custom titles on the dateplots. That would I think be a feature addition.
